git-dit.1.md

```diff
@@ -95,6 +96,223 @@ the former one is especially useful for patch-sets.
 Discussion messages will be created using `git-dit-reply` as empty commits,
 preserving the parent commit's tree.
 
+Note that in any case the parentship of an initial message should only be
+considered a reference.
+The initial message will not be part of the branch it may refer to at some
+point, and git-dit tools will not advance a branch if a new issue is created.
+
+Each issue has a "haad" reference, which references an "upstream" state of an
+issue.
+Metadata, such as assignments and the issue's status, is accumulated from an
+issue's reference towards the issue's initial commit.
+Additionally, a maintainer may use the "head" reference to communicate
+acceptance of some discussion point.
+
+
+# WORKFLOWS
```

Comment:
It appears that this is the only open point. Can I have an ACK/additional input from you?